sdk/src/main/java/com/kin/ecosystem/marketplace/presenter/MarketplacePresenter.java

```diff
@@ -195,14 +201,28 @@ private void release() {
 
 	@Override
 	public void getOffers() {
-		this.offerRepository.getOffers(new KinCallbackAdapter<OfferList>() {
+		this.offerRepository.getOffers(new KinCallback<OfferList>() {
 			@Override
 			public void onResponse(OfferList offerList) {
+				setupEmptyItemView();
```

Comment:
should we update unit tests to make sure setupEmptyItemView is called once and only once?